### ISSUE TITLE
git machete does not understand https://github.com remote URLs without trailing .git suffix

### ISSUE THREAD

**Issue by cspotcode** [PawelLipski/git-machete#724]
# Overview

`git machete github` does not understand that URLs of the following form are github URLs:

`https://github.com/myorg/myrepo`

It only understands if a `.git` suffix is added.

`https://github.com/myorg/myrepo.git`

Similar to #531.

## Expected behavior/command output

`git machete github` commands understand github URLs without trailing .git suffix.

## Actual behavior/command output

```
Remotes are defined for this repository, but none of them corresponds to GitHub (see git remote -v for details).
```

## `git` and `git machete` versions

```
git --version
git version 2.37.1
git machete --version
git machete version 3.13.0
```

**Comment by PawelLipski** [comment]
Whoops indeed good spot! @amalota pls check all 4 URL patterns within `git_machete.github.GITHUB_REMOTE_PATTERNS`... whether each of them can work without `.git` suffix (esp. the SSH URLs)

**Comment by PawelLipski** [comment]
To be released soon as a part of v3.13.1
